ui/pages/providers/[id].js

```diff
@@ -210,7 +208,8 @@ export default function ProvidersEditDetails() {
                 Client Secret
               </label>
               <input
-                type='password'
+                type='search'
```

Comment:
This change will disable password manager integration such as autofill and saving to the password manager but unhides the client secret while it's being edited

Reply:
Keeping this as `type=password` for now